content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md

```diff
@@ -221,6 +221,9 @@ option. Your cluster requirements may need a different configuration.
    {{< note >}}
    The `kubeadm-certs` Secret and decryption key expire after two hours.
    {{< /note >}}
+   {{< note >}}
+   The certificate key is a hex encoded string that is an AES key of size 32 bytes.
+   {{< /note >}}
```

Comment:
I agree it looks odd to have two notes. I agree that putting the suggested note in this PR as plain text before the existing note

Reply:
Sure thing !

Reply:
@SaumyaBhushan I think there is a misunderstanding here.
The secret and the key expires after two hours. This is something we want to highlight to the users, thus a `note` shortcode is warranted.
However, the key is a AES key of 32 bytes ... this doesn't have to be a note. I may and may not care about this details, because it doesn't hurt if I am not aware of it.
Make sense?

Reply:
So ... I am sorry if I was not clear in previous comments.
The note decorator/shortcode is supposed to be applied on the expiration statement because that is sometime not well received and causing failures.
However, the size of the key is not that important and it doesn't warrants a note. 

```
  The certificate key is a hex encoded string that is an AES key of size 32 bytes.

  {{< note >}}
  The `kubeadm-certs` Secret and decryption key expire after two hours.
  {{< /note >}}
```

Reply:
@SaumyaBhushan would you be able to update this again? ^^^^

Reply:
Sure , I can . Sorry I didnt get time on week days to correct it .